.github/workflows/merge-to-stage.js

```diff
@@ -199,6 +199,14 @@ const openStageToMainPR = async () => {
       base: PROD,
       body,
     });
+
+    await github.rest.issues.createComment({
+      owner,
+      repo,
+      issue_number: number,
+      body: 'Testing can start @adobecom/miq-sot @adobecom/bacom-sot @adobecom/homepage-sot @adobecom/creative-cloud-sot @adobecom/document-cloud-sot',
```

Comment:
Should we move the teams into an array constant to be able to update it easily if ever needed?